go.mod

```diff
@@ -125,3 +125,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/openshift/api v0.0.0-20220906163444-2df055c101a3 => github.com/suleymanakbas91/api v0.0.0-20220928121657-76624ee4fd58
```

Comment:
This will be removed once https://github.com/openshift/api/pull/1222 gets merged.